### ISSUE TITLE
[Bug]: a global gsudo install fails to update correctly

### ISSUE THREAD

**Issue by yonizaf** [ScoopInstaller/Main#4989]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

gsudo

### Expected/Current Behaviour

I had gsudo installed as a global install. version 2.0.4. New availabe version is 2.3.0.
Running `scoop update -g *`, I  expect it to update everything including gsudo.
in practice, running the above command, gsudo is updated, then an error shows up:
`Installing PowerShell modules globally is not implemented! `
any packages after that are not installed.

checking after the fact, the actual gsudo version is updated, but not listed as updated through `scoop status`.

### Steps to Reproduce

```console
> scoop status
Scoop is up to date.

Name           Installed Version Latest Version Missing Dependencies Info
----           ----------------- -------------- -------------------- ----
gsudo          2.0.4             2.3.0
mingit-busybox 2.39.1.windows.1  2.41.0.3

> scoop update -g *
gsudo: 2.0.4 -> 2.3.0 (global)
mingit-busybox: 2.39.1.windows.1 -> 2.41.0.3 (global)
Updating 2 outdated apps:
Updating 'gsudo' (2.0.4 -> 2.3.0)
Downloading new version
gsudo.setup.x64.msi (2.2 MB) [================================================================================] 100%
Checking hash of gsudo.setup.x64.msi ... ok.
Uninstalling 'gsudo' (2.0.4)
Removing shim 'sudo.shim'.
Removing shim 'sudo.exe'.
Unlinking C:\ProgramData\scoop\apps\gsudo\current
Installing 'gsudo' (2.3.0) [64bit] from main bucket
Loading gsudo.setup.x64.msi from cache
Extracting gsudo.setup.x64.msi ... done.
Linking C:\ProgramData\scoop\apps\gsudo\current => C:\ProgramData\scoop\apps\gsudo\2.3.0
Creating shim for 'sudo'.
Installing PowerShell modules globally is not implemented!

> scoop status
Scoop is up to date.

Name           Installed Version Latest Version Missing Dependencies Info
----           ----------------- -------------- -------------------- ----
gsudo          2.0.4             2.3.0
mingit-busybox 2.39.1.windows.1  2.41.0.3

> gsudo -V
gsudo v2.3.0 (Branch.tags-v2.3.0.Sha.c7c3c9380a70bd004c4a3bebe9f54b86d836afbd)
Copyright(c) 2019-2022 Gerardo Grignoli and GitHub contributors
```

### Scoop and Buckets Version

```console
> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
3360010ed (HEAD -> master, origin/master, origin/HEAD) ungoogled-chromium: Update to version 115.0.5790.110-1.1

'main' bucket:
6b737796e (HEAD -> master, origin/master, origin/HEAD) dprint: Update to version 0.40.0

'versions' bucket:
8de631b03 (HEAD -> master, origin/master, origin/HEAD) sharex-dev: Update to version 15.0.1.197
```

### Scoop Config

```console
> scoop config

last_update         scoop_branch scoop_repo
-----------         ------------ ----------
31/07/2023 09:22:10 master       https://github.com/ScoopInstaller/Scoop
```

### PowerShell Version

```console
> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.2.13
PSEdition                      Core
GitCommitId                    7.2.13
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by HUMORCE** [comment]
As the output message, when installing `main/gsudo` globally, it's powershell module will not installed, but that does not affect it's executable file.

`Installing PowerShell modules globally is not implemented!` is just an hints of unimplemented function, it does not mean that the installation failed.

You should create a feature request in the Scoop Core Repo instead of a bug report for gsudo.

**Comment by yonizaf** [comment]
why a feature request for a bug?

the problem here is twofold:
1. gsudo is updated on disk, but scoop is unaware of it and keeps reporting the old version.
2. the update script stops running and does not update the next packages on  the list.

**Comment by HUMORCE** [comment]
> gsudo is updated on disk, but scoop is unaware of it and keeps reporting the old version.

Sorry, I did not test this part, I only noticed that the executable was not affected.

> the update script stops running and does not update the next packages on the list.

Yes, you are right. The message will abort installations and updates flow same time.

> please reopen this or at least give a better response.

This issue will not be reopened:

- The unimplemented feature affects all manifests with PSMODULE, not just `main/gsudo`
- You have already created issue about this in Scoop Core Repo for now

Thanks for your report.
